homeassistant/components/opengarage/button.py

```diff
@@ -0,0 +1,89 @@
+"""OpenGarage button."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from typing import Any, cast
+
+import opengarage
```

Comment:
I think 
```suggestion
from opengarage import OpenGarage
```
might work better as you're also using `opengarage` as variable name at some places